packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -98,24 +115,67 @@ class CodeSizeBodyState extends State<CodeSizeBody>
               controller: _tabController,
               tabs: tabs,
             ),
+            Row(
+              children: [
+                if (!isSnapView) _buildDiffTreeTypeDropdown(),
```

Comment:
probably better to check that it is the diff view here in case we end up with other views that also are "not the snap view". We can rename to `isDiffView` everywhere instead